### PR TITLE
First round of Migration and Seeds completed.

### PR DESCRIPTION
**Pull request by AlexJoeb**
# Description

First round of migrations and seeds completed. Migrations imitates the first round database structure that we foresee needing. Addition/reductions to columns may occur in the future. Database also is seeded with basic information. 

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Change Status

- [ ] Complete, tested, ready to review and merge
- [x] Complete, but not tested (may need new tests)
- [ ] Incomplete/work-in-progress, PR is for discussion/feedback

# Has This Been Tested?

- [ ] Yes
- - [ ] What was tested?
- [x] No

# Checklist
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] My code has been reviewed and commented by at least two peers
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [x] There are no merge conflicts

### CODE REVIEW

**Review by DrakeAlia — DISMISSED** [review]

**Review by DrakeAlia — APPROVED** [review]

**Review by Carrillo9Ana — COMMENTED** [review]
😄 What steps will I have to do so I'm up to date with your databases and backend overall? It would be great to see that in the description, in case a teammate wants to see your code locally. e.g. do knex rollbak etc.

File: data/seeds/001_users.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  var lastName = faker.name.lastName();
+  return {
+    email: idx == 0 ? 'lambda@labs.com' : faker.internet.email(),
+    first_name: idx == 0 ? 'Lambda' : `${firstName}`,
+    last_name: idx == 0 ? 'School' : `${lastName}`,
+    school: `Lambda School`,
+    bg_username: idx == 0 ? 'Lambda-School' : `${firstName}-${lastName}`,
+    phone: `${faker.phone.phoneNumber()}`,
+    password: bc.hashSync(
+      idx == 0 ? 'password' : `${firstName}${faker.random * 500}`,
```
Comment (antonyk) [inline data/seeds/001_users.js:15]: Are you seeding with a random pass here? And if so, how will you be able to use this user afterwards?

File: data/migrations/20200625220949_create-profile.js [hunk 2]
```diff
@@ -1,15 +1,61 @@
...
+      tbl.increments();
+      tbl.string('datetime');
+      tbl.string('duration');
+      // RoomID and UserID
+      tbl
+        .integer('user_id')
+        .unsigned()
+        .notNullable()
+        .references('id')
+        .inTable('')
```
Comment (antonyk) [inline data/migrations/20200625220949_create-profile.js:41]: table name seems to be missing here... shouldn't it be 'users'?

**Review by antonyk — COMMENTED** [review]
Good start!

**Review by antonyk — APPROVED** [review]

**Pull request merged**